.github/config.yml

```diff
@@ -15,10 +15,14 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-   &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Thank you for contributing to the Layer5 community! :tada:
+  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Thank you for contributing to the Layer5 community! :tada:
   \
   \
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery/master/.github/welcome/Layer5-celebration.png)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/94157520/231786474-cafb4373-52ee-4b02-b06e-87376721c74a.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/94157520/231786445-45bf80c4-d214-40ab-a98f-c7a5f42015c5.svg">
+  <img alt="Meshery Logo" src="">
```

Comment:
I thought the `src` here should not be blank. I believe we should use the resources from the repo.

Reply:
I have made the changes as suggested. Please review it